src/compas_fab/robots/trajectory.py

```diff
@@ -21,17 +22,19 @@ class JointTrajectoryPoint(Configuration):
     Trajectory points are defined either as *values + velocities and
     accelerations*, or as *values + efforts*.
 
-    Attributes
+    Parameters
     ----------
-    values : :obj:`list` of :obj:`float`
-        Joint values expressed in radians or meters, depending on the respective type.
-    types : :obj:`list` of :class:`compas.robots.Joint.TYPE`
-        Joint types, e.g. a list of :class:`compas.robots.Joint.REVOLUTE` for revolute joints.
-    velocities : :obj:`list` of :obj:`float`
+    values : :class:`list` of :class:`float`
```

Comment:
What's the difference between `:class:` and `:obj`? I was under the impression the built-in objects are supposed to be documented with `:obj:`, but I might be wrong.

Reply:
From [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects):

> ###  :py:class:
> Reference a class; a dotted name may be used.
>
> ### :py:obj:
> Reference an object of unspecified type. Useful e.g. as the [default_role](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role).
>
> _New in version 0.4._

My reasoning was that `:class:` is more specific. But then again, are the builtins classes? I'm very open to use `obj` here. I actually had to start googling the difference between class, object and instance because of these.. (As I understand it both an instance and a class is an object..?)

Reply:
After some more research, yeah let's go with `:obj:`, it seems to be the standard.

Reply:
👍